src/CSI.IBTA.DB/EmployerUsers.sql

```diff
@@ -0,0 +1,9 @@
+﻿CREATE TABLE [dbo].[EmployerUsers]
+(
+	[Id] INT NOT NULL PRIMARY KEY,
```

Comment:
Should not it be autoincrement property? [I did not check other tables, hopefully they also auto](https://learn.microsoft.com/en-us/sql/t-sql/statements/create-table-transact-sql-identity-property?view=sql-server-ver16)

I would advise check other tables and ensure that we use autoincrement IDs when applicable.

Reply:
Will do in a couple of minutes